IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -97,7 +97,9 @@ + (BOOL)isCertAuthInProgress
 
 + (void)resetHandler
 {
+#if TARGET_OS_IPHONE && !MSID_EXCLUDE_SYSTEMWV
     s_certAuthInProgress = NO;
+#endif
```

Comment:
Do we need `#if !MSID_EXCLUDE_SYSTEMWV` for line 83 and 93? Seems `s_certAuthInProgress` is only defined when !MSID_EXCLUDE_SYSTEMWV

Reply:
Yes, this is needed because the declaration of the variable itself is wrapped in the macro as well, so there is actually a build error when MSID_EXCLUDE_SYSTEMWV is set without this

Reply:
And regarding this:

> for my own learning, why do we need #if defined TARGET_OS_VISION && TARGET_OS_VISION but not #if TARGET_OS_VISION? :)

This is for backwards compatibility with older XCode versions that may not have visionOS support